### PR TITLE
HOCS-2655 Date not showing on document preview (MSG file)

### PR DESCRIPTION
**Pull request by agraham19**
* Update msg parsing library
* Unit test msg parsing

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java [hunk 1]
```diff
@@ -13,14 +9,23 @@
 import com.itextpdf.tool.xml.ElementList;
 import com.itextpdf.tool.xml.XMLWorkerHelper;
 import com.itextpdf.tool.xml.exceptions.RuntimeWorkerException;
+import com.sun.star.util.DateTime;
```
Comment (robertdeniszczyc2) [inline src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java:12]: This is unused import
```suggestion
```

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java [hunk 2]
```diff
@@ -13,14 +9,23 @@
...
+import org.simplejavamail.outlookmessageparser.model.OutlookAttachment;
+import org.simplejavamail.outlookmessageparser.model.OutlookFileAttachment;
+import org.simplejavamail.outlookmessageparser.model.OutlookMessage;
 import org.springframework.stereotype.Component;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.text.MessageFormat;
+import java.time.LocalDateTime;
```
Comment (robertdeniszczyc2) [inline src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java:25]: This is unused import
```suggestion
```

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java [hunk 3]
```diff
@@ -13,14 +9,23 @@
...
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.TimeZone;
```
Comment (robertdeniszczyc2) [inline src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java:28]: This is unused import
```suggestion
```

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java [hunk 4]
```diff
@@ -133,6 +152,11 @@ private boolean parseElements(Document document, MsgContents contents) {
         try {
             ElementList elements = XMLWorkerHelper.parseToElementList(contents.getBodyHTML(), null);
             document.add(new Paragraph(""));
+
+            if(elements.isEmpty()) {
```
Comment (robertdeniszczyc2) [inline src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java:156]:
```suggestion
            if (elements.isEmpty()) {
```

**Review by robertdeniszczyc2 — COMMENTED** [review]
Just a couple trivial suggestions to pull in, rest looks good to me!

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java (hunk 2 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java]: I have made this change.

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java (hunk 4 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java]: I have made this change.

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java (hunk 1 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java]: I have made this change.

File: src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java (hunk 3 above)
Comment (agraham19) [reply src/main/java/uk/gov/digital/ho/hocs/document/MSGDocumentConverter.java]: I have made this change.

**Review by robertdeniszczyc2 — APPROVED** [review]

**Pull request merged**